.eslintrc.js

```diff
@@ -22,15 +22,15 @@ module.exports = {
     "one-var": ["error", "never"],
     "prefer-arrow-callback": "error",
     "prefer-const": "error",
-    "prettier/prettier": "error",
+    "prettier/prettier": ["error", { trailingComma: "es5" }],
```

Comment:
Please don't put the setting here, it breaks Prettier editor integrations that rely on Prettier's config file.

Reply:
Ah, I didn't consider that. I had originally used `.prettierrc`, but the version of `eslint-plugin-prettier` we had didn't take it into account, so I had to upgrade that first. Fixed in https://github.com/prettier/prettier/pull/3492/commits/ed0b21faaa59db4ef162fa1e9341fe9644257cb7.